sonar/modules/documents/cli/urn.py

```diff
@@ -64,3 +72,59 @@ def register_urn_identifiers():
     else:
         click.secho('No urns found to register', fg='yellow')
 
+
+@urn.command('snl-upload-files')
+@click.argument('timespan', default=1)
+@with_appcontext
+def snl_upload_files(timespan):
```

Comment:
I do not understand this. Usually we give the document pid or better the URN to put the corresponding files on the SNL server. We do not put all record having a registered URN.